specutils/io/default_loaders/sdss.py

```diff
@@ -3,6 +3,7 @@
 
 .. _spec: https://data.sdss.org/datamodel/files/BOSS_SPECTRO_REDUX/RUN2D/spectra/PLATE4/spec.html
 """
+from math import log
```

Comment:
```suggestion
```

Just a nit-pick: I think we should try to stick to numpy functions since we use those for all our other math needs.